.docker/entrypoint/spec/matchers/invoke_matcher.rb

```diff
@@ -28,9 +28,11 @@ def method_missing(name, *args, &block)
     self
   end
 
+  # rubocop:disable Lint/MissingSuper
```

Comment:
Codacy found an issue: [Unnecessary disabling of `Lint/MissingSuper` (unknown cop).](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5971046)